internal/applicationsnapshot/input.go

```diff
@@ -176,3 +210,79 @@ func readSnapshotSource(input []byte) (app.SnapshotSpec, error) {
 	log.Debugf("Read application snapshot from file %s", input)
 	return file, nil
 }
+
+func (*remoteClient) Get(ref name.Reference) (*remote.Descriptor, error) {
+	desc, err := remote.Get(ref)
```

Comment:
`return remote.Get(ref)` ?

Same for the `Index` and `clientFromContext` methods below.

Reply:
Nice! I changed `Get` and `Index`.  `clientFromContext` couldn't be changed since Go compiler doesn't allow that saying ctx.Value(key).(client) only returns a single value